SPEC/OBJECT.md

```diff
@@ -164,23 +156,21 @@ A great source of [examples][] can be found in the tests for this API.
 
 - `enc`, the encoding of multihash (base58, base64, etc), if any.
 
-`callback` must follow `function (err, links) {}` signature, where `err` is an error if the operation was not successful and `links` is an Array of [DAGLink](https://github.com/vijayee/js-ipfs-merkle-dag/blob/master/src/dag-node.js#L199-L203) objects.
+**Returns**
 
-If no `callback` is passed, a [promise][] is returned.
+| Type | Description |
+| -------- | -------- |
+| `Promise<Array>` | An Array of [DAGLink](https://github.com/vijayee/js-ipfs-merkle-dag/blob/master/src/dag-node.js#L199-L203) objects |
```

Comment:
Please update to https://github.com/ipld/js-ipld-dag-pb/blob/master/src/dag-link/dagLink.js